tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -104,6 +106,30 @@ def policy_manager(testerchain, escrow):
     return contract, dispatcher
 
 
+@pytest.fixture()
+def overseer(testerchain, escrow):
```

Comment:
Did we change the name from `overseer` to something else?

Reply:
Yes, it seems I forgot to change this. Good catch!

Reply:
✅